src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -69,6 +80,13 @@
 						infReferenceGeneProductInst.addAttributeValue(species, speciesInst);
 						String referenceGeneProductSource = homologueSource.equals("ENSP") ? "ENSEMBL:" : "UniProt:";
 						infReferenceGeneProductInst.setAttributeValue(_displayName, referenceGeneProductSource + homologueId);
+
+						// If the species is C. elegans, add each matching gene name to the 'geneName' attribute of the RGP.
+						if (isCelegans) {
+							for (String wormbaseGeneName : wormbaseGeneNames) {
+								infReferenceGeneProductInst.addAttributeValue(geneName, wormbaseGeneName);
```

Comment:
Is `geneName` not in `ReactomeJavaConstants`? If not, it _can_ be added.

Reply:
In Orthoinference we have statically imported `ReactomeJavaConstants`, so it is actually being called here. We might want to consider reverting that though, since it is the only step doing it this way and it could be a bit confusing for someone looking at the code in the future

Reply:
Ah ok. I don't normally use static imports. That would explain why I missed it.